haxxx/validation/deploy-validation.yaml

```
@@ -0,0 +1,57 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+---
+- name: Packit-service validation deployment
+  hosts: all
+  vars:
+    # path with needed secrets (github, sentry, copr)
+    path_to_secrets: ""
+    project: packit-service-validation
+    host: https://open.paas.redhat.com
+    # oc login <the above host value>, oc whoami -t
+    # OR via Openshift web GUI: click on your login in top right corner, 'Copy Login Command', take the part after --token=
+    api_key: ""
+    validate_certs: no
+
+  tasks:
+    - name: Deploy secrets
+      # https://docs.ansible.com/k8s_module.html
+      k8s:
+        namespace: "{{ project }}"
+        definition: "{{ item }}"
+        host: "{{ host }}"
+        api_key: "{{ api_key }}"
+        verify_ssl: "{{ validate_certs }}"
```

See https://github.com/packit-service/deployment/pull/83

```suggestion
        validate_certs: "{{ validate_certs }}"
```